core/runtime/common/trie_storage_provider_impl.cpp

```diff
@@ -119,7 +103,7 @@ namespace kagome::runtime {
     if (persistent_batch_ != nullptr) {
       return persistent_batch_->commit();
     }
-    return trie_storage_->getRootHash();
+    return Error::NO_BATCH;
```

Comment:
This change is most likely the reason for the crash of the genesis test in w3f/polkadot-spec#108.